src/porepy/models/constitutive_laws.py

```diff
@@ -726,19 +619,17 @@ def fluid_density(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
         return rho
 
 
-class FluidMobility:
+class FluidMobility(PorePyModel):
     """Class for fluid mobility and its discretization in flow problems."""
 
     fluid_viscosity: Callable[[list[pp.Grid]], pp.ad.Operator]
     """Function that returns the fluid viscosity. Normally provided by a mixin of
-    instance :class:`~porepy.models.VariableMixin`.
+    instance TODO.
 
     """
-    mobility_keyword: str
-    """Keyword for the discretization of the mobility. Normally provided by a mixin of
-    instance :class:`~porepy.models.SolutionStrategy`.
 
-    """
+    mobility_keyword: str
+    """TODO"""
```

Comment:
Not sure I understand this. Let's discuss shortly in person.

Reply:
Fixed